### DISCUSSION TITLE
I found this error even i changed the test network

### DISCUSSION THREAD

**Discussion by Jatin40** [ConsenSys-archive/truffle#6069] [General Q&A]
PS C:\Users\Jatin\OneDrive - nsut.ac.in\Desktop\Truffle\web> truffle migrate --network Goerli

Compiling your contracts...
===========================
> Compiling .\contracts\Migrations.sol
> Compiling .\contracts\demo.sol
> Artifacts written to C:\Users\Jatin\OneDrive - nsut.ac.in\Desktop\Truffle\web\build\contracts
> Compiled successfully using:
   - solc: 0.8.0+commit.c7dfd78e.Emscripten.clang

Starting migrations...
======================
> Network name:    'Goerli'
> Network id:      5
> Block gas limit: 30000000 (0x1c9c380)

1_initial_migration copy.js
===========================

   Deploying 'Migrations'
   ----------------------
   > transaction hash:    0x8122323dff9c122bafc64ace4f469cb945cfe05b866ef68840907ad3f8902db2
   ⠏ Blocks: 1            Seconds: 18
C:\Users\Jatin\node_modules\eth-block-tracker\src\polling.js:51
        const newErr = new Error(`PollingBlockTracker - encountered an error while attempting to update latest block:\n${err.stack}`)
                       ^
Error: PollingBlockTracker - encountered an error while attempting to update latest block:
undefined
    at PollingBlockTracker._performSync (C:\Users\Jatin\node_modules\eth-block-tracker\src\polling.js:51:24)   
    at processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by FlaviuWeb3** [comment]
Hi @Jatin40 could you please share with me the repository for your contract so I can try to reproduce this?
